contrib/nvidia/nvidia.go

```diff
@@ -181,3 +185,23 @@ func WithRequiredCUDAVersion(major, minor int) Opts {
 		return nil
 	}
 }
+
+// WithOCIHookPath sets the hook path for the binary
+func WithOCIHookPath(path string) Opts {
+	return func(c *config) error {
+		c.OCIHookPath = path
+		return nil
+	}
+}
+
+// WithLookupOCIHookPath sets the hook path for the binary via a binary name
+func WithLookupOCIHookPath(name string) Opts {
```

Comment:
Any reason not to do `LookPath` all the time and avoid 2 options?

Reply:
options are cheap

Reply:
Yeah, I got it, I'm just considering the PBKAC. But I guess options don't cost a thing